### ISSUE TITLE
Issues with image faker on fixtues

### ISSUE THREAD

**Issue by bobman38** [montessori-ressources/web#38]
Groumf I have issues with the image faker when running the fixture setup...

```

   > purging database
   > loading App\DataFixtures\AppFixtures
   > loading App\DataFixtures\UserFixtures
   > loading App\DataFixtures\CardFixtures

In MimeTypeGuesser.php line 119:

  The file "" does not exist


doctrine:fixtures:load [--append] [--group GROUP] [--em EM] [--shard SHARD] [--purge-with-truncate] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--
no-interaction] [-e|--env ENV] [--no-debug] [--] <command>
```

_Originally posted by @bobman38 in https://github.com/montessori-ressources/web/pull/37#issuecomment-500237069_

Maybe better to rely on real images from @allan ? What do you think ?

**Comment by bobman38** [comment]
Will be fixed by #39 using local resources,

**Comment by StaudenmannM** [comment]
indeed fixed
